pkg/interface/src/logic/subscription/global.ts

```diff
@@ -43,12 +43,25 @@ export default class GlobalSubscription extends BaseSubscription<StoreState> {
     groups: []
   };
   start() {
-    this.subscribe('/all', 'invite-store');
+    this.scry('/all', 'invite-store');
+    this.subscribe('/updates', 'invite-store');
+
+    this.scry('/groups', 'group-store');
     this.subscribe('/groups', 'group-store');
+
     this.subscribe('/primary', 'contact-view');
-    this.subscribe('/all', 'metadata-store');
+
+    this.scry('/associations', 'metadata-store');
+    this.subscribe('/updates', 'metadata-store');
+
+    this.scry('/credentials', 's3-store');
+    this.scry('/configuration', 's3-store');
     this.subscribe('/all', 's3-store');
```

Comment:
We should probably just write an /updates subscription ... or maybe just tell it to scry from s3-store after setting it in `profile`, because it's unlikely to change very often.